itou/employee_record/models.py

```diff
@@ -94,6 +95,33 @@ def set_asp_processing_information(self, code, label, archive):
         self._set_archived_json(archive)
 
 
+class EmployeeRecordTransition(enum.StrEnum):
+    READY = "ready"
+    SENT = "sent"
```

Comment:
```suggestion
    PREPARE = "prepare"
    SEND = "send"
```
pour coller aux autres valeurs et être un verbe ? ou `MARK_AS_READY` ? Voir garder l'existant `UPDATE_AS_READY` ?

Reply:
Je crois que ces transitions ont eu entre 3 et 5 noms différents en fonction des moments.

J'ai choisi la forme passive `"sent"` plutôt que `"send"` car ce n'est pas la transition qui fait l'action de l'envoi et que celui-ci est déjà fait au moment de l'appel de fonction.
Pour `"ready"`, c'est aussi un verbe ;) donc avec un peu de mauvaise foi je me suis dit "c'est pas si pire, le mot colle à ce qu'on veux dire et on garde le même vocabulaire".

A voir ce que ça vaut encore demain mais peut-être un truc de ce style sinon ? :shrug: 
```suggestion
    SEND = "send"
    SENT = "sent"
```

Reply:
Va pour le verbe `ready` et le renommage `update_as_ready` -> `ready`:+1: 

Et OK également pour ne pas appeler `update_as_sent` `send` car effectivement l'envoi a lieu hors-transition.
Mais je préfère `update_as_sent` plutôt que `sent` (ou `log_batch_information` ? vu que c'est tout ce que fait la transition :sweat_smile: )

Reply:
-1 pour `log_batch_information`, ce n'est pas l'action qu'on veux tracer, ces informations sont aussi enregistrées dans d'autre transitions, et l'appel à la fonction dans la transition va disparaître puisque ça va être porté par le `TransitionLog`.

-0 pour `update_as_sent`, mais OK pour ne pas garder `sent`.
C'est surtout le préfixe `update_as_` qui me pose problème car je trouve qu'on descend dans le technique à dire pourquoi on appelle la fonction plutôt que dire ce qui c'est passé, alors peut-être `sent_to_asp` ? Ou `transferred` ?